java/arcs/core/data/Ttl.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+/**  A class containing retention policy information. */
+data class Ttl(
+    val count: Int?,
+    val units: Units?
+) {
+    enum class Units {
+        Minute,
+        Hour,
+        Day
+    }
+
+    fun isInfinite(): Boolean = this == Ttl.Infinite
+
```

Comment:
My only comment is the equals() method that the data class by default won't be true for something like Ttl(60, Minute) == Ttl(1, Hour) 

I think I would override these, convert everything to seconds, and compare seconds.

Reply:
done, PTAL